extensions/resource-deployment/src/services/tools/azCliTool.ts

```diff
@@ -99,12 +99,11 @@ const win32InstallationCommands = [
 	},
 	{
 		comment: localize('resourceDeployment.AziCli.DisplayingInstallationLog', "displaying the installation log …"),
-		command: `type AzureCliInstall.log | findstr /i /v /c:"cached product context" | findstr /i /v /c:"has no eligible binary patches" `,
```

Comment:
This was a pre-existing bug where the file name of log file was changed in the command that was generating the log but not in the command that was outputing the log to the outputChannel